### PR TITLE
Replace hardcoded pi values with M_PI

### PR DESCRIPTION
**Pull request by r-burns**

### CODE REVIEW

**Review by ribbanya — COMMENTED** [review]
Did you search the binary for exact matches, or only the C source?

**Comment by r-burns** [comment]
Just the source. There are probably more occurrences, these are just the ones I noticed.

**Comment by ribbanya** [comment]
Do you want to do a full pass in this PR? We could use a `.define` in GAS and your `#define` in C.

File: src/melee/ft/chara/ftMario/ftMario_SpecialHi.c [hunk 1]
```diff
@@ -51,7 +51,7 @@ void ftMario_SpecialAirHi_Anim(HSD_GObj* gobj)
 }
 
 #define DEGREES_TO_RADIANS (3.14159265359f / 180.0f)
```
Comment (ribbanya) [inline src/melee/ft/chara/ftMario/ftMario_SpecialHi.c:53]: Is this instance different bitwise from the define?
```suggestion
#define DEGREES_TO_RADIANS (3.14159265359f / 180.0f)
```

**Review by ribbanya — CHANGES_REQUESTED** [review]

**Comment by r-burns** [comment]
I'm just doing the no-brainers here, there are some nontrivial cases that will require some discussion - like the f32-typed `3.1415927410125732f` values, which are probably from a macro or an inline function.

File: src/melee/ft/chara/ftMario/ftMario_SpecialHi.c (hunk 1 above)
Comment (r-burns) [reply src/melee/ft/chara/ftMario/ftMario_SpecialHi.c:53]: Yeah not sure what to do with these, should probably consolidate all the deg-to-rad conversion constants as well. This is float-typed so it needs to be part of a macro or an inline.

**Review by ribbanya — APPROVED** [review]

**Pull request merged**